mlir/utils/performance/perfRunner.py

```diff
@@ -1212,15 +1166,16 @@ def benchmarkFusionKernels(test_dir, paths: Paths, arch, numCU, tuningDb: MaybeT
         # Find the best perf_config
         bestPerf =""
         if tuningDb:
-            if (arch, testVector) in tuningDb:
-                bestPerf = tuningDb[arch, testVector]
+            configStr = config.toCommandLine()
+            if (arch, configStr) in tuningDb:
+                bestPerf = tuningDb[arch, configStr]
                 config.setPerfConfig(bestPerf)
             else: # Tuning DB present but doesn't contain config, add a NaN entry
                 if not testVector in perfResults:
                     oneEntry = config.tableEntry(np.nan)
-                    oneEntry['FileName'] = filename
                     oneEntry['MLIR TFlops'] = np.nan
                     oneEntry['Fusion/MLIR'] = np.nan
+                    oneEntry['FileName'] = filename
```

Comment:
Add the columns in the same order that they are in the running case.  Doesn't matter in practice, but it bugged me in diffs.